app/build.gradle

```diff
@@ -32,10 +32,16 @@ android {
 
 dependencies {
     implementation project(":libndt7")
-    implementation "com.android.support:appcompat-v7:$support_library_version"
+    //implementation "com.android.support:appcompat-v7:$support_library_version"
```

Comment:
In general, I'm happier without commented out code. Unless there's a reason. Example:

```C
  // The following is only useful when debugging. Uncomment if needed.
  (void)fprintf(stderr, "userID: %s\n", userID);
```